apps/examples/dtls_server/dtls_server_main.c

```diff
@@ -40,15 +40,35 @@
  * Included Files
  ****************************************************************************/
 
-#include "tls/config.h"
+#include <tinyara/config.h>
```

Comment:
You're right, but it has no harm to include it just for safety. Anyway, I will fix it.